src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,3 +1,5 @@
 databaseChangeLog:
   - include:
-      file: db/changelog/changes/01-create-books-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/01-create-books-table.yaml
+  - include:
+      file: db/changelog/changes/02-create-users-table.yaml
```

Comment:
```suggestion
      file: db/changelog/changes/02-create-users-table.yaml

```
don't forget to leave last line empty
Always review your PR, and add one line where you see this red sign
![image](https://github.com/kacper-kukula/book-store/assets/49767049/38ac1c4e-80b6-41f7-b1b9-542d2d85f348)